nutrinova-api/.editorconfig

```diff
@@ -110,4 +110,9 @@ dotnet_diagnostic.IDE0065.severity = none
 dotnet_diagnostic.IDE0009.severity = none
 dotnet_diagnostic.SA1101.severity = none
 # IDE0005: Remove unused usings
-dotnet_diagnostic.IDE0005.severity = error
\ No newline at end of file
+dotnet_diagnostic.IDE0005.severity = error
+
+[NutrinovaDbContext.cs]
+dotnet_diagnostic.CS8618.severity = none
+[NutrinovaDbContext.cs]
```

Comment:
you can remove the second [NutrinovaDbContext.cs ]  I think

Reply:
done